lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -4794,7 +4868,7 @@ ByteCodeGenerator::GetLdSlotOp(Scope *scope, int envIndex, Js::RegSlot scopeLoca
     return op;
 }
 
-void ByteCodeGenerator::EmitPropLoad(Js::RegSlot lhsLocation, Symbol *sym, IdentPtr pid, FuncInfo *funcInfo)
+void ByteCodeGenerator::EmitPropLoad(Js::RegSlot lhsLocation, Symbol *sym, IdentPtr pid, FuncInfo *funcInfo, bool useParamScope)
```

Comment:
Is it not possible to figure out that the scope we're loading from is the param scope and dispense with the extra flag parameter?